show_ligand_interactions.py

```diff
@@ -0,0 +1,43 @@
+#show_ligand_interactions v.1.0
+# author: Thomas Evangelidis, 2019
+# License: BSD-2-Clause
+
+from pymol import cmd, util
+
+def show_ligand_interactions(recsel="not hetatm", ligsel="hetatm", cuttof=5):
```

Comment:
do you want to rename `cuttof` of `cutoff`? Looks like a typo to me.

Reply:
yes, thank you!